lib/ui/src/modules/ui/components/search_box.js

```diff
@@ -102,25 +101,23 @@ class SearchBox extends React.Component {
         isOpen={showSearchBox}
         onAfterOpen={this.onModalOpen}
         onRequestClose={onClose}
-        className={css({
-          fontSize: theme.mainTextColor,
-          fontFamily: theme.mainTextFace,
-          color: theme.mainTextColor,
-        })}
-        overlayClassName={css({
-          position: 'fixed',
-          height: '100vh',
-          width: '100vw',
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-          top: 0,
-          left: 0,
-          right: 0,
-          bottom: 0,
-          background: theme.overlayBackground,
-          zIndex: 1,
-        })}
+        style={{
+          overlay: {
+            height: '100vh',
+            width: '100vw',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            background: theme.overlayBackground,
+            zIndex: 1,
+          },
+          content: {
+            fontSize: theme.mainTextColor,
+            fontFamily: theme.mainTextFace,
+            color: theme.mainTextColor,
+          },
+        }}
+        className="tmp" // This exists to make the Modal discard it's default styles for content
```

Comment:
Can we just override the styles that we don't want to have?

Reply:
There are too many for content.

Reply:
Content has a lot of default styles. This was a cleaner way to do it.